app/grandchallenge/algorithms/templates/algorithms/algorithmimage_detail.html

```diff
@@ -108,15 +108,15 @@ <h2>Algorithm Container Image</h2>
             </span>
         </dd>
 
-        <dt>Requires GPU</dt>
+        <dt>Requires GPU Type</dt>
         <dd>
-            <span class="badge badge-{{ object.requires_gpu|yesno:'success,danger' }}">
-                {{ object.requires_gpu }}
+            <span class="badge badge-{% if object.algorithm.job_requires_gpu_type %}info{% else %}danger{% endif %}">
+                {% if object.algorithm.job_requires_gpu_type %}{{ object.algorithm.job_requires_gpu_type }}{% else %}No GPU{% endif %}
```

Comment:
Not tested this but I think it is more Django-y:

```suggestion
                {{ object.algorithm.get_job_requires_gpu_type_display }}
```